Fix: Responsividade da Hero e seção "Como você pode ajudar"

BrianTKK opened the pull request.

BrianTKK committed:
Fix: Responsividade da Hero e seção "Como você pode ajudar"

amimarinho reviewed (changes requested):
Brian, como alinhamos, essa alteração no caminho não funciona quando testamos no github pages, então vamos voltar para ./ como estava, tudo bem?

BrianTKK committed:
fix: Corrige links quebrados adicionando prefixo de caminho relativo

Adiciona './' aos atributos href dos links para resolver problemas de caminho
relativo e garantir que os links apontem para os recursos corretos.

BrianTKK commented:
Corrigido links quebrados adicionando prefixo de caminho href

BrianTKK committed:
Fix 2 caminho dos arquivos JS

BrianTKK committed:
Fix erro JS não carregando projetos

BrianTKK committed:
Fix adicionando arquivos atualizados da branch atualizada

amimarinho committed:
Merge branch 'main' into main
